### PR TITLE
Updated the Create an Element guide

### PR DESCRIPTION
**Pull request by bvhme**

### CODE REVIEW

File: creating-elements/readme.md [hunk 1]
```diff
@@ -1,58 +1,51 @@
...
+## 3. Create a new element
+
+So you need to build a new element. Think about with which other elements the new element should interact with. What information does the new element need from them and what can it give back? 🗣
+
+Create a HTML file with a name that describes the the element very well and at least one `-`, or hyphen, in it, for instance `user-chip`, `project-name`, `user-login-form` or `helicopter-landing-pad`. 
+
+To get started you could copy and paste the content of `seed-element.html` into your empty file to setup your new element.
 
-So you need to build a new element. Think about with which other elements the new element should interact. What information does the new element need from them and what can it give back? 🗣
+Every element should be able to work in isolation and you should build/test it in isolation. Create a demo file, insert your element, give it the relevant data and test your file.
```
Comment (MatthijsMentink) [inline creating-elements/readme.md]: maybe explain what a demo file, or link to a description. Now its kinda bomb dropping
.

File: creating-elements/readme.md [hunk 2]
```diff
@@ -1,58 +1,51 @@
...
+
+You are likely to think about building functionality that actually falls outside the scope of a good element. Don't build the element only for the achievable you are working on though, think bigger and make something reusable.🚀
+
+## 3. Create a new element
+
+So you need to build a new element. Think about with which other elements the new element should interact with. What information does the new element need from them and what can it give back? 🗣
+
+Create a HTML file with a name that describes the the element very well and at least one `-`, or hyphen, in it, for instance `user-chip`, `project-name`, `user-login-form` or `helicopter-landing-pad`. 
+
+To get started you could copy and paste the content of `seed-element.html` into your empty file to setup your new element.
```
Comment (MatthijsMentink) [inline creating-elements/readme.md]: where can i find this seed-element.html?

**Comment by bvhme** [comment]
Updated! @MatthijsMentink

**Comment by MatthijsMentink** [comment]
NICE!

**Pull request merged**